### ISSUE TITLE
Labkit ui interface sleep in middle and update not to 100%

### ISSUE THREAD

**Issue by rmd13** [juglab/labkit-ui#90]
When I run Labkit to segment large images (512x512x18), the segment results update only for 70% of tiles then stop update especially when the labkit window is not at fous. 
To increase this number, I have to click on the image, switching z-sices, etc.

**Comment by maarzt** [comment]
Yes, the UI by default only calculates the segmentation for the part of the image that is currently visible.
In the Labkit menu there an entry: "Segmentation > Calculate entire Segmentation Result" which you may run to pre-calculate the enter segmentation.
